### PR TITLE
add startup script at scripts/startup.sh

### PR DESCRIPTION
**Pull request by windwaker1121**
I have written the startup script.

### CODE REVIEW

File: scripts/startup.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#Install nodejs
+sudo apt-get install -y npm curl
+sudo curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+sudo npm install
+
```
Comment (tz70s) [inline scripts/startup.sh]: `install-deps.sh` above.

File: scripts/startup.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  update "../.."
+else
+  echo "You are in the wrong dir, nexcom-project replacement failed."
+fi
+
+#Start and enable daemon
+sudo systemctl daemon-reload
+sudo systemctl enable nexcom.service
+sudo systemctl start nexcom.service
+
```
Comment (tz70s) [inline scripts/startup.sh]: I see, the deployment can be along with update, but the commands on daemon of `first deployment` and `update` may be different.

File: scripts/startup.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+echo "Installation of postgreSQL-9.6 sussece!"
+
+#scheme Construction
+echo "Create database in the postgres..."
+sudo -u postgres psql -c "create database nexcom;"
+echo "Load the dump file from ../database/nexcom_v2_layout"
+sudo -u postgres psql nexcom < ../database/nexcom_v2_layout
+echo "Construction of scheme sussece!"
+
+exit 0
```
Comment (tz70s) [inline scripts/startup.sh]: These goes to another database script.

Comment (windwaker1121) [reply scripts/startup.sh]: OK, I'm working on it.
Should I use the startup.sh to call all the split scripts, or just let them keep split to execute？

Comment (tz70s) [reply scripts/startup.sh]: Sure, you can keep the integration script stayed. Name it as `start-up-once.sh`.

File: scripts/service-backend.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  sudo cp ./scripts/nexcom.service /etc/systemd/system/nexcom.service
+  echo "succesed!"
+elif [ ! -e "../../nexcom-project/scripts" ]; then
+  echo "You executed service-backend.sh at wrong dir."
+elif [ ! -e "/etc/systemd/system/" ]; then
+  echo "'/etc/systemd/system/' dir not exist, copy daemon failed." 
+else
+  echo "copy nexcom.service to /etc/systemd/system/"
+  sudo cp ./nexcom.service /etc/systemd/system/nexcom.service
+  echo "succesed!"
```
Comment (tz70s) [inline scripts/service-backend.sh]: A little more modification,
`succesed` -> `success`

File: scripts/service-backend.sh [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+fi
+
+#Start and enable daemon
+sudo systemctl reload-or-restart nexcom.service
+
+echo "Update and Deployment in the service-backend.sh Done."
```
Comment (tz70s) [inline scripts/service-backend.sh]: `Update and deploy done.`

File: scripts/service-backend.sh (hunk 4 above)
Comment (windwaker1121) [reply scripts/service-backend.sh]: OK, I already fixed it, thanks.

File: scripts/service-backend.sh [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  update "../.."
+else
+  echo "You are in the wrong dir, nexcom-project replacement failed."
+fi
+
+#nexcom.service deployment
+if [ -e "./scripts" ]; then
+  echo "copy nexcom.service to /etc/systemd/system/"
+  sudo cp ./scripts/nexcom.service /etc/systemd/system/nexcom.service
+  echo "succesed!"
```
Comment (tz70s) [inline scripts/service-backend.sh]: @windwaker1121 Here, too, lol.

File: scripts/service-backend.sh [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  if [ $backupexi == "exist" ]; then
+    echo "Backup old file to nexcom-project.backup.."
+    sudo rm -r /root/nexcom-project.backup
+    sudo mv /root/nexcom-project /root/nexcom-project.backup
+  elif [ $fileexist == "exist" ]; then
+    echo "Backup old file to nexcom-project.backup."
+    sudo mv /root/nexcom-project /root/nexcom-project.backup
+  fi
+  sudo cp -a $path"/nexcom-project" /root/
+  echo "succesed!"
```
Comment (tz70s) [inline scripts/service-backend.sh]: Here

File: scripts/service-backend.sh (hunk 6 above)
Comment (windwaker1121) [reply scripts/service-backend.sh]: Oh, my.... OK, I will check all the scripts again.

**Comment by tz70s** [comment]
Great work! Big thanks!

**Pull request merged**